src/methods.jl

```diff
@@ -164,32 +164,16 @@ One can also do weighted fits:
 
 See also `classes`, `support`.
 """
-function Dist.pdf(
-    d::UnivariateFinite{S,V,R,P},
-    cv::CategoricalValue,
-) where {S,V,R,P}
-    return get(d.prob_given_ref, int(cv), zero(P))
-end
-Dist.pdf(d::UnivariateFinite{S,V}, c::V) where {S,V} = _pdf(d, c)
-Dist.pdf(::UnivariateFinite{S,V}, ::Missing) where {S,V} = missing
-
-# Avoid method ambiguity errors with Dist >= 0.24
-Dist.pdf(d::UnivariateFinite{S,V}, c::V) where {S,V<:Real} = _pdf(d, c)
+Dist.pdf(::UnivariateFinite, ::Missing) = missing
 
-function _pdf(d::UnivariateFinite, c)
+function pdf(d::UnivariateFinite, c)
```

Comment:
Don't you mean `Dict.pdf` here?

Reply:
yes - I was just writing the idea without testing. Now, given you approve the direction, I will make a PR out of it.